src/js/video.js

```diff
@@ -542,264 +485,43 @@ videojs.addLanguage = function(code, data) {
-
-/**
- * Format seconds as a time string, H:MM:SS or M:SS
- * Supplying a guide (in seconds) will force a number of leading zeros
- * to cover the length of the guide
- *
- * @borrows format-time:formatTime as videojs.formatTime
- */
 videojs.formatTime = formatTime;
-
-/**
- * Replaces format-time with a custom implementation, to be used in place of the default.
- *
- * @borrows format-time:setFormatTime as videojs.setFormatTime
- *
- * @method setFormatTime
- *
- * @param {Function} customFn
- *        A custom format-time function which will be called with the current time and guide (in seconds) as arguments.
- *        Passed fn should return a string.
- */
 videojs.setFormatTime = setFormatTime;
-
-/**
- * Resets format-time to the default implementation.
- *
- * @borrows format-time:resetFormatTime as videojs.resetFormatTime
- *
- * @method resetFormatTime
- */
 videojs.resetFormatTime = resetFormatTime;
-
-/**
- * Resolve and parse the elements of a URL
- *
- * @borrows url:parseUrl as videojs.parseUrl
- *
- */
 videojs.parseUrl = Url.parseUrl;
-
-/**
- * Returns whether the url passed is a cross domain request or not.
- *
- * @borrows url:isCrossOrigin as videojs.isCrossOrigin
- */
 videojs.isCrossOrigin = Url.isCrossOrigin;
-
-/**
- * Event target class.
- *
- * @borrows EventTarget as videojs.EventTarget
- */
 videojs.EventTarget = EventTarget;
-
-/**
- * Add an event listener to element
- * It stores the handler function in a separate cache object
- * and adds a generic handler to the element's event,
- * along with a unique id (guid) to the element.
- *
- * @borrows events:on as videojs.on
- */
 videojs.on = Events.on;
-
-/**
- * Trigger a listener only once for an event
- *
- * @borrows events:one as videojs.one
- */
 videojs.one = Events.one;
-
-/**
- * Removes event listeners from an element
- *
- * @borrows events:off as videojs.off
- */
 videojs.off = Events.off;
-
-/**
- * Trigger an event for an element
- *
- * @borrows events:trigger as videojs.trigger
- */
 videojs.trigger = Events.trigger;
 
 /**
- * A cross-browser XMLHttpRequest wrapper. Here's a simple example:
+ * A cross-browser XMLHttpRequest wrapper.
  *
- * @param {Object} options
- *        settings for the request.
+ * @function
+ * @param    {Object} options
+ *           Settings for the request.
  *
- * @return {XMLHttpRequest|XDomainRequest}
- *         The request object.
+ * @return   {XMLHttpRequest|XDomainRequest}
+ *           The request object.
  *
- * @see https://github.com/Raynos/xhr
+ * @see      https://github.com/Raynos/xhr
  */
 videojs.xhr = xhr;
 
-/**
- * TextTrack class
- *
- * @borrows TextTrack as videojs.TextTrack
- */
 videojs.TextTrack = TextTrack;
-
-/**
- * export the AudioTrack class so that source handlers can create
- * AudioTracks and then add them to the players AudioTrackList
- *
- * @borrows AudioTrack as videojs.AudioTrack
- */
 videojs.AudioTrack = AudioTrack;
-
-/**
```

Comment:
we should verify that we didn't lose any info all of these removals.

Reply:
➕

Reply:
Makes sense.

Reply:
The only things I know we do lose are the top-level DOM function aliases that are deprecated. I may be able to restore them without manually documenting each, but it seemed like a better thing to have the non-deprecated API documented.